lib/phoenix_live_view/route.ex

```diff
@@ -29,8 +29,11 @@ defmodule Phoenix.LiveView.Route do
   end
 
   def live_link_info!(%Socket{} = socket, view, uri) do
+    %{private: %{live_session_name: session_name, live_session_vsn: session_version}} = socket
+
     case live_link_info(socket.endpoint, socket.router, uri) do
-      {:internal, %Route{view: ^view} = route} ->
+      {:internal,
+       %Route{view: ^view, live_session: %{name: ^session_name, vsn: ^session_version}} = route} ->
```

Comment:
For completeness, I don’t think the version could ever differ for a patch, as it is the same version.

Reply:
In such case the websocket would have to reconnect. If it doesn't reconnect, then it would still be the same vsn.

Reply:
The `vsn` is purely serverside. If a deployment happened, the user would have reconnected. (The only scenario where something might be different - I think - is when there was a hot code upgrade.)